development/prepare-code-for-review.md

```diff
@@ -12,7 +12,13 @@ If it's not obvious, it can also be helpful to include instructions on how to te
 
 ## Review Your Own Branch
 
-When you have your initial version for the proposed change working, look at your changes once again and ask yourself if anything could be improved before marking it "Ready for Review".
+When you have your initial version for the proposed change working, look at your changes once again and ask yourself if anything could be improved before marking it "Ready for Review". Be sure to check that:
+
+✅️ Your branch adheres to the elementary [code style guide](https://docs.elementary.io/develop/writing-apps/code-style)
+
+✅️ New Terminal warnings are not introduced
```

Comment:
I'm not technically the right person to suggest that, but I had to re-read the sentence as I missed the `not` while reading, so I propose something like:
```suggestion
✅️ No New Terminal warnings are introduced
```

Reply:
Makes sense to me :)